internal/pkg/agent/cmd/run.go

```diff
@@ -545,6 +547,7 @@ func tryDelayEnroll(ctx context.Context, logger *logger.Logger, cfg *configurati
 		&options,
 		paths.ConfigFile(),
 		store,
+		fleetgateway.RequestBackoff, // for delayed enroll, we want to use m
```

Comment:
I think this comment got cut off or something. `use m`?

Reply:
Oops, fixed now.